fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseFormatType.java

```diff
@@ -1,20 +1,10 @@
-// Copyright 2016 Yahoo Inc.
+// Copyright 2017 Yahoo Inc.
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 package com.yahoo.bard.webservice.web;
 
-import java.util.Locale;
+public interface ResponseFormatType {
```

Comment:
This needs some docs, since it looks like it'll be a core, foundational piece of content negotiation going forward (which is a good thing!)